godtools/Views/TractElements/TractRoot.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  TractRoot.swift
+//  godtools
+//
+//  Created by Devserker on 4/27/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SWXMLHash
+
+//  NOTES ABOUT THE COMPONENT
+//  * This component must always be initialized with: init(data: XMLIndexer, withMaxHeight height: CGFloat)
+
+class TractRoot: BaseTractElement {
+    
+    override func setupView(properties: Dictionary<String, Any>) {
```

Comment:
`Dictionary<String, Any>` can be declared as: `[Dictionary: Any]`

reference: https://github.com/raywenderlich/swift-style-guide#syntactic-sugar